x/plans/keeper/migrations.go

```diff
@@ -4,6 +4,9 @@ import (
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	v3 "github.com/lavanet/lava/x/plans/migrations/v3"
```

Comment:
I think you meant "v4" here (the subdir names the target version); 
But we should combine this anyhow into v2 -> v3 migration, so this ends up being ok :)

Reply:
I opened a subdir for **old** protobuf versions. So I changed this to v2